go.mod

```diff
@@ -62,9 +62,9 @@ require (
 	golang.org/x/text v0.27.0
 	gopkg.in/natefinch/lumberjack.v2 v2.2.1
 	gopkg.in/yaml.v3 v3.0.1
-	k8s.io/api v0.29.3
-	k8s.io/apimachinery v0.29.3
-	k8s.io/client-go v0.29.3
+	k8s.io/api v0.32.3
+	k8s.io/apimachinery v0.32.3
+	k8s.io/client-go v0.32.1
```

Comment:
`k8s.io/client-go` goes up to `v0.32.7`, I think it’s ok to use something newer. I haven’t checked the other ones.

Reply:
I want to use same version of k8s client-go which is part of https://github.com/openshift/client-go/blob/release-4.19/go.mod or in openshift-installer to be compatible with.